packages/shared/routes/dashboard/staking/views/StakingSummary.svelte

```diff
@@ -107,9 +107,7 @@
                     on:mouseenter={toggleTooltip}
                     on:mouseleave={toggleTooltip}
                 >
-                    {#if isPartiallyStaked}
-                        <Icon icon="exclamation" classes="fill-current text-yellow-600" />
-                    {/if}
+                    <Icon icon="exclamation" classes="fill-current text-yellow-600" />
```

Comment:
It's not visible in this diff, but there is a Svelte conditional rendering block also checking `isPartiallyStaked`, so it is redundant to have here.